apps/app/src/react/App.tsx

```diff
@@ -191,6 +195,35 @@ export const App = observer(function App() {
 		}
 	}
 
+	const allButtonActions = useMemoComputedObject(() => {
```

Comment:
Would it be possible to move this calculation into the [ProjectStore](https://github.com/SuperFlyTV/SuperConductor/blob/develop/apps/app/src/react/mobx/ProjectStore.ts#L45)?
I think that it might be a better place to do that, to follow the existing way we do these types of calculations.

Reply:
I'm not sure how that would be done, as this calculation also requires data from the RundownsStore and we don't currently support inter-store operations like this from within a store.

Reply:
Ok, I'm not sure what's best practice here either